src/App/PropertyLinks.cpp

```diff
@@ -71,6 +72,15 @@ void PropertyLinkBase::setAllowExternal(bool allow) {
     setFlag(LinkAllowExternal,allow);
 }
 
+void PropertyLinkBase::setSilentRestore(bool allow) {
+    setFlag(LinkSilentRestore,allow);
+}
+
+void PropertyLinkBase::setReturnNewElement(bool enable)
+{
+    setFlag(LinkNewElement, enable);
+}

```

Comment:
Two methods right next to each other with different placement of opening `{`. This file could really use clang-format.

Reply:
It could, but I'm not wading into the wholesale reformatting war right now.  This local change has been fixed.